TopMovies/View/MovieVC/MovieVCProps.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  MovieVCProps.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 25.03.2021.
+//
+
+import UIKit
+
+struct MovieVCProps {
+  let posterPlaceholderImage: UIImage
+  let isDownloadingInProgress: Bool
+  let isAdultImage: UIImage?
+  let isNewImage: UIImage?
+  let posterURL: URL?
+  let genresLabelTextArray: [String]
+  let productionCompaniesLogosURL: [URL?]
+  let titleLabeltext,
+      taglineLabelText,
+      overviewLabelText,
+      ratingAndVotesLabelText,
+      budgetLabelText,
+      statusLabelText,
+      releaseDateLabelText,
+      runtimeLabelText: String
```

Comment:
Let's add `let` and `string` for all of them to make it easier to change property type.
Let's say you'd want to change `ratingAndVotesLabelText` to `String?`.
You will write `let ratingAndVotesLabelText: String?` before `titleLabelText` or after `runtimeLabelText.

But if you do that, you will break existing init, so you'll have to split that construction...